contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -80,4 +99,14 @@ contract OETHHarvesterSimple is Strategizable {
     {
         IERC20(_asset).safeTransfer(strategistAddr, _amount);
     }
+
+    function setDripper(address _dripper) external onlyGovernorOrStrategist {
```

Comment:
This should just be onlyGovernor

Reply:
Changed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2363/commits/17fcfb0dd2100dec77790dd37760704b380e3a0e